app/src/components/ProfileEdit/QuestionPage.tsx

```diff
@@ -52,122 +76,170 @@ export const QuestionPage = (props: Props) => {
         if (!groups[groupI][subgroupI]) groups[groupI][subgroupI] = []
         groups[groupI][subgroupI].push(state.questions[i])
     }
-
+    
     function setAnswer(index: number, answer: any) {
-        var state2 = {...state}
+
+        const indexQuesitons = [0, 1, 2, 3, 4, 5, 6, 7, 8] //for modalv
```

Comment:
Good job, Hassen. The modal is working well.
You can make it more dynamic by adding a property to the QuestionType interface. That way it will work for other question sets we pass in as JSON. The property can be called showstopper with the value being the answer as a string.